src/Model.php

```diff
@@ -776,33 +776,64 @@ public function setMulti(array $fields)
      *
      * @return mixed
      */
-    public function get(string $field = null)
+    public function get(string $fieldName = null)
     {
-        if ($field === null) {
+        if ($fieldName === null) {
             // Collect list of eligible fields
             $data = [];
             if ($this->only_fields) {
                 // collect data for actual fields
-                foreach ($this->only_fields as $field) {
-                    $data[$field] = $this->get($field);
+                foreach ($this->only_fields as $fieldName) {
+                    $data[$fieldName] = $this->get($fieldName);
                 }
             } else {
                 // get all fields
-                foreach ($this->getFields() as $field => $f) {
-                    $data[$field] = $this->get($field);
+                foreach ($this->getFields() as $fieldName => $field) {
+                    $data[$fieldName] = $this->get($fieldName);
                 }
             }
 
             return $data;
         }
 
-        $this->checkOnlyFieldsField($field);
+        // if field is not existing maybe it is a traversing link or aggregate function
+        if (!$this->hasField($fieldName)) {
+            // alias for count aggregate, consistent with Model scopes
+            if ($fieldName === '#') {
+                $fieldName = 'count(*)';
+            }
+
+            // traverses relationships and returns value from a referenced model, e.g contact/company/name
+            if (str_contains($fieldName, '/')) {
```

Comment:
can be deduplicate this, it seems on a several places now

Reply:
`Reference::isTraversingLink`?

Reply:
I mean the whole code, not only str_contains...

Reply:
I looked now more into the code - `get(something else than field name)` is brutal antipattern - it seems super handy, but in reality it will hit db again and again - I am quite sure to not add this - `Model::get()` should return only loaded data, it is up too you to define them first.